src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -106,6 +107,10 @@ def algorithm(self, parameters):
         get information from wmbs, workqueue and local couch
         """
         try:
+            config_path = os.path.join(os.environ.get('config', ''), 'config.py')
+            if not os.path.exists(config_path):
+                config_path = '/data/srv/wmagent/current/config/wmagent/config.py'
```

Comment:
this may be a problem for jenkins. Maybe we should check if the default path exists as well. If it does NOT, then we don't even load the config and we don't save.
Moreover, when you save the config you are also persisting it on disk (I believe...). So I'd only save it if it really changed.